packages/ui/package.json

```diff
@@ -10,7 +10,9 @@
   ],
   "main": "app/main.js",
   "scripts": {
-    "postinstall": "npm link ../daemon",
+    "postinstall": "run-s electron-rebuild link-daemon",
+    "link-daemon": "npm link --force ../daemon",
+    "electron-rebuild": "electron-rebuild",
```

Comment:
Following this: https://www.electronjs.org/docs/tutorial/using-native-node-modules#installing-modules-and-rebuilding-for-electron